geometry_msgs/PoseStamped pose    		# Position of the goal
---
bool ack 			# acknowledged information